src/components/Reports/FinancialAccountsReport/AccountTransactions/AccountTransactions.test.tsx

```diff
@@ -30,117 +24,130 @@ jest.mock('react-csv/lib/core');
 
 const accountListId = 'account-list-1';
 const financialAccountId = 'financialAccountId';
-const defaultSearchTerm = '';
-const setActiveFilters = jest.fn();
 const setPanelOpen = jest.fn();
-const defaultActiveFilters = {};
 const defaultRouter = {
   query: { accountListId },
   isReady: true,
 };
 const mutationSpy = jest.fn();
 
 interface ComponentsProps {
-  activeFilters?: FinancialAccountTransactionFilters;
-  searchTerm?: string;
   router?: object;
```

Comment:
Here are the types you can use.

Although, it could make the tests simpler if you had `Components` accept a `filters` and `searchTerm` props that `Components` uses to build the `router` object. That would let us make the individual tests more concise and not need as much spreading.

```suggestion
  router?: Partial<NextRouter>;
```